### ISSUE TITLE
Can't connect to pfunified_api in latest devel

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#3057]
Probably since its been switched to localhost for its bind

```
[root@julien-cluster-1 pf]# netstat -nlp | grep 8080
tcp6       0      0 ::1:8080                :::*                    LISTEN      15660/perl          
[root@julien-cluster-1 pf]# curl 127.0.0.1:8080
curl: (7) Failed connect to 127.0.0.1:8080; Connection refused
```

**Comment by jrouzierinverse** [comment]
Fix by 5ec80922ec9f69d9d02b52492b86cd1a7a804474

**Comment by jrouzierinverse** [comment]
The problem was I assumed that localhost included ipv4
